### ISSUE TITLE
Document how to contribute to AlertManager and AlertManagerConfig CRD

### ISSUE THREAD

**Issue by slashpai** [prometheus-operator/prometheus-operator#6429]
### Component(s)

AlertManager, AlertManagerConfig

### What is missing? Please describe.

I feel like it's confusing for new contributors on how to add a new feature to AlertManager as it needs to be added to AlertManager config (static config) as well as AlertManagerConfig CRD. It will make it easier to point to documentation rather than repeating same on all issues.

May be https://github.com/prometheus-operator/prometheus-operator/blob/main/CONTRIBUTING.md is a good place to add that?

### Describe alternatives you've considered.

NA

### Environment Information.

NA

**Comment by Ayush9026** [comment]
@slashpai mam I will try to solve this.

**Comment by Ayush9026** [comment]
Yes mam you are right https://github.com/prometheus-operator/prometheus-operator/blob/main/CONTRIBUTING.md is a good place to add that? 
I will open PR for this.

**Comment by codeknight03** [comment]
Hey @Ayush9026 are you still working on this ?

**Comment by slashpai** [comment]
@codeknight03 Please feel free to work on the issue :)
